diff -ur Llvm-21.1.5/cmake/modules/HandleLLVMOptions.cmake Llvm-21.1.5-new/cmake/modules/HandleLLVMOptions.cmake
--- Llvm-21.1.5/cmake/modules/HandleLLVMOptions.cmake	2025-11-03 09:35:47.000000000 +0000
+++ Llvm-21.1.5-new/cmake/modules/HandleLLVMOptions.cmake	2025-11-26 19:37:51.511375880 +0000
@@ -1423,7 +1423,7 @@
   endif()
 endif()
 
-set(LLVM_THIRD_PARTY_DIR  ${CMAKE_CURRENT_SOURCE_DIR}/../third-party CACHE STRING
+set(LLVM_THIRD_PARTY_DIR  ${CMAKE_CURRENT_SOURCE_DIR}/llvm-third-party-21.1.5.src CACHE STRING
     "Directory containing third party software used by LLVM (e.g. googletest)")
 
 set(LLVM_UNITTEST_LINK_FLAGS "" CACHE STRING
diff -ur Llvm-21.1.5/CMakeLists.txt Llvm-21.1.5-new/CMakeLists.txt
--- Llvm-21.1.5/CMakeLists.txt	2025-11-03 09:35:47.000000000 +0000
+++ Llvm-21.1.5-new/CMakeLists.txt	2025-11-26 19:37:51.510548047 +0000
@@ -3,7 +3,7 @@
 
 include(CMakeDependentOption)
 
-set(LLVM_COMMON_CMAKE_UTILS ${CMAKE_CURRENT_SOURCE_DIR}/../cmake)
+set(LLVM_COMMON_CMAKE_UTILS ${CMAKE_CURRENT_SOURCE_DIR}/llvm-cmake-21.1.5.src)
 include(${LLVM_COMMON_CMAKE_UTILS}/Modules/CMakePolicy.cmake
   NO_POLICY_SCOPE)
 
